src/styles/blocks/forms.scss

```diff
@@ -0,0 +1,26 @@
+.input,
+.select,
+.textarea {
```

Comment:
It is better to add some dedicated class, like `form-control`, and use it on form elements